src/go/rpk/pkg/cli/container/start.go

```diff
@@ -140,18 +148,26 @@ func startCluster(
 		return nil
 	}
 
-	fmt.Println("Checking for a local image...")
-	present, checkErr := common.CheckIfImgPresent(c, image)
-	if checkErr != nil {
-		fmt.Printf("Error trying to list local images: %v\n", err)
-	}
-	if !present {
-		// If the image isn't present locally, try to pull it.
-		fmt.Println("Downloading latest version of redpanda")
+	if pull {
+		fmt.Println("Force pulling image...")
```

Comment:
When would a user need to force pull? Is this to update the latest version, even if some local image exists?

Reply:
Yeah, realistically with the way we do tags, this only applies to `latest` (as it happens, also the default), but it's a nice thing to have to avoid needing a separate `docker` call. As we're already well within the realm of wrapping docker, it doesn't feel too out of place either

If we were ever to change our tagging to have e.g. `v23.2`, then it would also be of use there.